Updated Unit to 1.32.0.

oxpa opened the pull request.

oxpa committed:
Updated Unit to 1.32.0.

tippexs reviewed (commented):
Will this configure the old `wasm` module only? The new module should be configured like mentioned here: https://unit.nginx.org/howto/source/#modules-webassembly

thresheek commented:
Yeah, we need to adapt packaging to include the new wasm module here too. I'll work on it.

thresheek committed:
Updated njs to 0.8.2.

thresheek committed:
Updated wasm package to include wasm-wasi-component.

This also requires a small patch to unit source code to fix FTBFS on an
older macOS version still used in github actions CI.

thresheek committed:
Updated php-embed to 8.3.3.

thresheek commented:
Now it finally looks good.  There seems to be a couple of test failures: python 2 on ubuntu, and ruby on both ubuntu and macos;  I'll deal with it in the subsequent PRs.
